contracts/protocol/facets/ExchangeHandlerFacet.sol

```diff
@@ -196,7 +196,10 @@ contract ExchangeHandlerFacet is IBosonExchangeHandler, BuyerBase, DisputeBase {
             Condition storage condition = fetchCondition(groupId);
 
             // Make sure condition is not SpecificToken as it is not supported for preminted offers
-            require(condition.method != EvaluationMethod.SpecificToken, CANNOT_COMMIT);
+            require(
+                condition.method == EvaluationMethod.Threshold && condition.tokenType != TokenType.MultiToken,
```

Comment:
Why is `MultiToken` not allowed for preminted offers?

Reply:
because caller must provide a additional `tokenId` parameter

Reply:
Referring to my other comment (in `validateCondition`), I think we should allow `length` and `tokenId` to be non-zero for `EvaluationMethod.Threshold`.

In that case, we could support both cases:
- when a range with length > 1, the user must specify which `tokenId` they want to use
- if the range has length 1, then it's unambiguous which `tokenId` is relevant and we can support preminted offers with multitoken condition.